www/%username/widgets/%type.spt

```diff
@@ -1,29 +1,38 @@
 from markupsafe import Markup
 
 from liberapay.utils import get_participant
+from liberapay.utils.i18n import Wrap
 
-A, B = Markup('<br><span style="font-size: 125%">'), Markup('</span>')
+span_open, span_close, br = Markup('<br><span style="font-size: 125%%">'), Markup('</span>'), Markup('<br>')
```

Comment:
Is %% correct CSS?

Reply:
We have to escape the '%' because this text will be formatted. Without the double %, an error is thrown because %" is not a valid format sequence.